[2025-01-25T14:08:00.562+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: openweather_data_lake.initialize_buckets manual__2025-01-25T14:07:59.299986+00:00 [queued]>
[2025-01-25T14:08:00.567+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: openweather_data_lake.initialize_buckets manual__2025-01-25T14:07:59.299986+00:00 [queued]>
[2025-01-25T14:08:00.567+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2025-01-25T14:08:00.575+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): initialize_buckets> on 2025-01-25 14:07:59.299986+00:00
[2025-01-25T14:08:00.584+0000] {standard_task_runner.py:57} INFO - Started process 932 to run task
[2025-01-25T14:08:00.588+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'openweather_data_lake', 'initialize_buckets', 'manual__2025-01-25T14:07:59.299986+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/openweather_data_lake.py', '--cfg-path', '/tmp/tmp14om5w5a']
[2025-01-25T14:08:00.591+0000] {standard_task_runner.py:85} INFO - Job 24: Subtask initialize_buckets
[2025-01-25T14:08:00.605+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-01-25T14:08:00.671+0000] {task_command.py:415} INFO - Running <TaskInstance: openweather_data_lake.initialize_buckets manual__2025-01-25T14:07:59.299986+00:00 [running]> on host 76d975d787c4
[2025-01-25T14:08:00.725+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='openweather_data_lake' AIRFLOW_CTX_TASK_ID='initialize_buckets' AIRFLOW_CTX_EXECUTION_DATE='2025-01-25T14:07:59.299986+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-25T14:07:59.299986+00:00'
[2025-01-25T14:08:00.955+0000] {logging_mixin.py:151} INFO - [INFO] Bucket 'raw' existe déjà.
[2025-01-25T14:08:00.956+0000] {logging_mixin.py:151} INFO - [INFO] Bucket 'staging' existe déjà.
[2025-01-25T14:08:00.956+0000] {logging_mixin.py:151} INFO - [INFO] Bucket 'curated' existe déjà.
[2025-01-25T14:08:00.957+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-01-25T14:08:00.967+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=openweather_data_lake, task_id=initialize_buckets, execution_date=20250125T140759, start_date=20250125T140800, end_date=20250125T140800
[2025-01-25T14:08:00.991+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-25T14:08:01.018+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
